examples/eap/docker-build/README.md

```diff
@@ -54,10 +54,10 @@ Technologies required to build and deploy this example
 
 * You have installed Helm. Please refer to [Installing Helm page](https://helm.sh/docs/intro/install/) to install Helm in your environment
 
-* You have installed the repository for the Helm charts for WildFly
+* You have installed the repository for the Helm charts for EAP 8
 
  ```
-helm repo add wildfly https://docs.wildfly.org/wildfly-charts/
+helm repo add jboss https://jbossas.github.io/eap-charts/
```

Comment:
should be `helm repo add jboss-eap https://jbossas.github.io/eap-charts`  as you refer to the Helm Chart with `jboss-eap/eap8`.

Reply:
Thank-you. Fixed.